app.js

```diff
@@ -16,11 +16,11 @@ if (process.env.NODE_ENV === 'development') {
   app.use(morgan('dev'));
 }
 
-app.use('/psy/users', authRouter, userRouter);
-app.use('/psy/doctors', authRouter, doctorRouter);
-app.use('/psy/admins', authRouter, adminRouter);
+app.use('/psy/users', userRouter);
+app.use('/psy/doctors', doctorRouter);
+app.use('/psy/admins', adminRouter);
 app.use('*', (req, res, next) => {
-  next(new AppError('this route is not defind ', 400));
+  next(new AppError('This route is not defind ', 404));
```

Comment:
page is not found is better, also make sure it's not there by default, 404 error